### DISCUSSION TITLE
"gaussian" in dpgen simplify

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/dpgen#1024] [Q&A]
There is a sentence of "fp_style can only be gaussian currently" in the introduction for dpgen simplify. Does this mean that "vasp" is not supported for this task? 

Thanks!

**Comment by HuangJiameng** [comment]
[fp_style](https://docs.deepmodeling.com/projects/dpgen/en/devel/simplify/simplify-jdata.html#argument:simplify_jdata/fp_style) can be either gaussian or vasp currently.
README will be discarded, please refer to https://docs.deepmodeling.com/projects/dpgen/en/devel/simplify/simplify.html
